go/tools/asthelpergen/integration/test_helpers.go

```diff
@@ -98,3 +100,8 @@ func Rewrite(node AST, pre, post ApplyFunc) AST {
 
 	return outer.AST
 }
+
+// ASTComparison is used to compare AST trees and override the normal comparison logic
+type ASTComparison interface {
+	ColNames(a, b *sqlparser.ColName) *bool
+}
```

Comment:
This is outdated. Is it being used at all?

Reply:
it is being passed around in the generated ast helpers in this package, but not actually invoked anywhere. I'll remove the sqlparser reference